src/com/returnhightnumbersolution/ReturnHightNumberSolution.java

```diff
@@ -0,0 +1,30 @@
+package com.returnhightnumbersolution;
+
+public class ReturnHightNumberSolution {
+
+    public static int returnHightNumber(){
+        int resultat=0;
+     int[] numbers=new int[]{3000,150,700,9,2000,500,1500,4,30000};
+        int k=0;
+        int i=0;
```

Comment:
to make it efficient, you only need one variable declared and please name it `max`